<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Beebview</class>
 <widget class="QMainWindow" name="Beebview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beebview</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>392</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>&amp;Mode</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCycleColour">
    <property name="title">
     <string>&amp;Cycle Colour</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionReportBug"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMode"/>
   <addaction name="menuCycleColour"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Online &amp;Help</string>
   </property>
  </action>
  <action name="actionReportBug">
   <property name="text">
    <string>&amp;Report a Bug</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
